### ISSUE TITLE
[VSCode-Proper] A11y_ViennaVsCodeExtension_ExplorerTAB_ScreenReader: Narrator is not reading the name of Toolbar(Outline).

### ISSUE THREAD

**Issue by greazer** [Microsoft/vscode#63254]
“Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

Environment Details:
Application Name: ViennaVsCodeExtension
Application Version: 1.28.0
Operating System: Windows10
Url:  https://marketplace.visualstudio.com/items?itemName=ms-toolsai.vscode-ai#overview

Repro Steps:
1) Launch VisualStudio Code.
2) Navigate to Explorer Tab hit Space bar key-> Navigate to Outline toolbar.
3) Start Narrator.
5) Verify what Narrator is reading for Outline toolbar.

Actual
Narrator is not reading the name of Outline toolbar as Narrator is just announcing it as toolbar.

Expected
Narrator should read the name of Outline toolbar.

User Impact
Screen reader user will not be able to understand which toolbar is focused.

Recommendation
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=1.3.1

MAS Reference
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D

**Comment by greazer** [comment]
[703402_A11y_ViennaVsCodeExtension_ExplorerTAB_ScreenReader.zip](https://github.com/Microsoft/vscode/files/2587476/703402_A11y_ViennaVsCodeExtension_ExplorerTAB_ScreenReader.zip)

**Comment by jrieken** [comment]
Cannot reproduce anymore, must have been fixed in the meantime.
